src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { Card, Avatar, Typography, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { greyScale } from '../../style-library/types/theme.helpers';
```

Comment:
Importing `greyScale` from '../../style-library/types/theme.helpers' couples this component to an internal 'style-library' path. That path may not exist in other projects or may be outside the component's encapsulation.

**Sugerencia:**
```typescript
Validate that `style-library` exists and is intended to be a project-wide theme helper. If you just need neutral grey colors, prefer `theme.palette` (e.g. `theme.palette.grey[500]`) or centralize shared tokens in a single theme file under `src/theme`.
```